WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -185,8 +146,8 @@ public void onResponse(JSONObject response) {
                         }
 
                         if (mSite.isJetpackConnected()) {
```

Comment:
I added it in here originally to prevent multiple error toasts when fetching. I'll move this block to `fetchRemoteData` and figure out a better way to handle the multiple error toasts in a future PR.